### PR TITLE
Update index page - Closes #789

### LINKED ISSUE #789: Update index page
- Update the index page to reflect the latest changes in the SDK v5
- Remove alpha stage disclaimer

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #789

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -54,6 +40,15 @@ The codebase is written entirely in JavaScript and TypeScript, which is highly b
...
 The Lisk SDK makes every effort to ensure developers are easily able to focus purely and simply on writing the code that is required for their own blockchain application and nothing else.
 
+=== Architecture overview
+
+The Lisk SDK operates on the Node.js runtime and consists primarily of an application framework (xref:{url_framework}[Lisk Framework]).
+This consists of a collection of libraries providing blockchain application functionalities (xref:{url_elements}[Lisk Elements]), and a powerful Command Line Interface (xref:{url_commander}[Lisk Commander]), which allows developers to manage a Lisk node instance and interact with a Lisk compatible network.
+
+image::architecture-3-level.png[Diagram]
+
+A detailed explanation about the underlying architecture of the Lisk framework is descirbed at xref:{url_framework}[]
```
Comment (chris529) [inline modules/ROOT/pages/index.adoc]:
```suggestion
A detailed explanation regarding the underlying architecture of the Lisk framework is described at xref:{url_framework}[]
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -85,21 +71,12 @@ Before running Lisk SDK, the following dependencies shown below need to be insta
...
-Further details on installing these dependencies can be found in the xref:{url_setup}[pre-installation setup guide].
-
-Please be aware that primarily it is necessary to create a database.
-The default database name is `lisk_dev`, so for the development purposes, a command `createdb lisk_dev --owner lisk` can be used for the set up.
-The default database user and password are `lisk` and `password`, they can be both changed in the xref:{url_config}[configuration] of the Lisk SDK.
-
 === Installation
 
-Installation of the Lisk Alpha SDK is straightforward and limited to getting a single NPM package, `lisk-sdk`, to your Node.js project as shown below:
+Installation of the Lisk SDK is straightforward and limited to getting a single NPM package, `lisk-sdk`, to your Node.js project as shown below:
```
Comment (chris529) [inline modules/ROOT/pages/index.adoc]:
```suggestion
Installation of the Lisk SDK is straightforward and only requires getting a single NPM package `lisk-sdk`, to your Node.js project as shown below:
```

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -144,7 +133,10 @@ The tutorials overview page provides an informative overview about all existing
 
 TIP: All code for the example apps that is used in the tutorials is also available in the {url_github_sdk_examples}[lisk-sdk-examples repository^] on GitHub.
 
-== Legal disclaimer
+=== The Lisk protocol
+
+The xref:{url_protocol}[Lisk protocol] is the set of rules followed by a blockchain created with the Lisk SDK using the default configuration.
+It contains various development-agnostic specifications about the Lisk SDK.
+
+It is a good point to look up certain topics to gain a deeper understanding, or to explore the SDK in a scientific direction.
```
Comment (chris529) [inline modules/ROOT/pages/index.adoc]:
```suggestion
At this point, it is recommended to look up certain topics in order to gain a deeper understanding, or to further explore the SDK in a scientific direction.
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a couple of minor suggestions 👍

**Review by mitsuaki-u — DISMISSED** [review]
LGTM other than @chris529 comments!

File: modules/ROOT/pages/index.adoc (hunk 3 above)
Comment (Tschakki) [reply modules/ROOT/pages/index.adoc]:
```suggestion
It is a good location to look up certain topics in order to gain a deeper understanding, or to further explore the SDK in a scientific direction.
```

**Pull request merged**